tools/model_tools/tests/open_model_zoo_test.py

```diff
@@ -0,0 +1,96 @@
+import numpy as np
+import unittest
+import os
+
+import openvino.model_zoo.open_model_zoo as omz
+
+from openvino.inference_engine import IECore
+
+class TestTopologies(unittest.TestCase):
+    def test_load_intel(self):
+        face_detection = omz.Model.download_model('face-detection-0200', precision='FP16-INT8')
+        xml_path = face_detection.model_path
+
+        self.assertTrue(os.path.exists(xml_path))
+
+        ie = IECore()
+        net = ie.read_network(xml_path)
+
+        input_name = next(iter(net.input_info))
+        output_name = next(iter(net.outputs))
+
+        self.assertEqual(net.input_info[input_name].input_data.shape, [1, 3, 256, 256])
+        self.assertEqual(net.outputs[output_name].shape, [1, 1, 200, 7])
+
+    def test_load_public(self):
+        model = omz.Model.download_model('colorization-v2', precision='FP32')
+        xml_path = model.model_path
+
+        self.assertTrue(os.path.exists(xml_path))
+
+        ie = IECore()
+        net = ie.read_network(xml_path)
+
+        input_name = next(iter(net.input_info))
+        output_name = next(iter(net.outputs))
+
+        self.assertEqual(net.input_info[input_name].input_data.shape, [1, 1, 256, 256])
+        self.assertEqual(net.outputs[output_name].shape, [1, 2, 256, 256])
+
+    def test_load_from_prtained(self):
```

Comment:
if that means test_load_from_pretrained, then better to name it as test_load_intel

Reply:
This is to test loading from locale file. `test_load_intel` already exists for downloading intel model.